File: WhichPharmaPortal/WhichPharma.Models/Client/RFQDetails.cs
using System.Collections.Generic;

namespace WhichPharmaPortal.Models.Client
{
    public class RFQDetails
    {
        public string RFQNumber { get; set; }
        public IEnumerable<RFQNote> Notes { get; set; }
        public IEnumerable<RFQSupplierDetails> SuppliersDetails { get; set; }
        public List<RFQCards> Cards { get; set; }
    }
}

File: WhichPharmaPortal/WhichPharmaPortal/ClientApp/src/models/RFQDetails.ts
import { RFQQuoteTableItem } from '../pages/RFQDetailsV2';
import { RFQNote } from './RFQNote';
import { RFQSupplierDetails } from './RFQSupplierDetails';

export interface RFQDetails {
    rfqNumber: string;
    notes: RFQNote[];
    suppliersDetails: RFQSupplierDetails[];
    cards?: RFQQuoteTableItem[];
}

export interface RFQDetailsResponse {
    result: RFQDetails[];
    timeInSeconds: number;
}
